659 dAPI intro roadmap (#662)
api3dao/vitepress-docs: docs/reference/dapis/understand/index.md

various decentralized applications such as DeFi lending, synthetic assets,
stable coins, derivatives, NFTs, and more.

## Values stored on-chain

API providers, owners of first-party Airnodes, provide the signed data used to

> store beacon values (via AirSeeker) on-chain as individual beacons. A dAPI's

Reviewer: "as individual beacons" seems redundant with "store beacon values" in this sentence. You could combine by having "...used to store individual beacon values on-chain."

Author: Done and thanks

> value is derived as an aggregated set of beacon values. dAPI values are held in

Reviewer: ```suggestion
value is in turn an aggregate of beacon values. dAPI values are held in
```

Author: Done

## The role of Airnode

Airnode is a flexible off-chain module that can support multiple protocols. Most
noticeably is its implementation of the request-response protocol (RRP) and data
feeds.

An Airnode is owned by an API provider and integrates itself along side their

> API operations. Airnode interfaces with these operations and gathers its signed

Reviewer: I'm not sure what "integrates itself along side" means. Perhaps something like "An Airnode is owned by an API provider and is used to call API provider endpoints to fetch and sign data. Airseeker uses the signed data..."

Author: Yes that was a run of the mouth there. Done

Reviewer: Looks like the old text meant to be replaced is still there: "An Airnode is owned by an API provider and integrates itself along side their API operations. Airnode interfaces with these operations and gathers its signed data at the request of Airseeker." Everything else looks good to me.

Author: @dcroote  I see the mistake. I added the new line but did not remove the old one.

> data at the request of AirSeeker. AirSeeker uses the signed data to determine if

Reviewer: Lowercase "s" in Airseeker (here and throughout).

Author: Ouch, got it

the deviation of a dAPI value (e.g., plus or minus 0.25%, 0.5%, 1%) warrants an
on-chain update of its beacon(s).

<img src="../assets/images/beacons-airnode.png" style="width:80%;">

> In the diagram above, companies ABC and XYZ both have an API operation (A and B)
> that is used to determine the value of the dAPI ZIL/USD . AirSeeker regularly

Reviewer: ```suggestion
In the diagram above, companies XYZ and ABC both provide ZIL/USD values, A and B, respectively, that are aggregated to determine the dAPI ZIL/USD value. Airseeker regularly
```

Author: For all the commas it does read nicely. You should be an English professor. Done

> checks the deviation of ZIL/USD (as well as other dAPIs) using the sign data
> from Airnode. AirSeeker will update the corresponding beacons behind ZIL/USD
> when deviation is detected.

Reviewer: ```suggestion
checks the deviation of ZIL/USD using the sign data
from these Airnodes. Airseeker will update the corresponding beacons behind ZIL/USD
when deviation is detected.
```

Author: Done

[Market](https://market.api3.org/dapis).

| Self-funded dAPIs                                                               | Managed dAPIs                                         |
| ------------------------------------------------------------------------------- | ----------------------------------------------------- |
| March 2023                                                                      | Fall 2023                                             |
| Single public [proxy contract](/reference/dapis/understand/proxy-contracts.md)  | Single public proxy contract                          |
| 1% deviation                                                                    | Multiple deviations<br/>(0.25%, 0.5%, 1%)             |
| 60 second [interval](/reference/dapis/understand/deviations.md#update-interval) | 30-60 second interval                                 |
| 24 hour [heartbeat](/reference/dapis/understand/deviations.md#heartbeat)        | 2 minute or 24 hour heartbeat                         |
| Sourced from a single<br/>data feed (beacon)                                    | Sourced from multiple<br/>data feeds (beacons)        |

> | Gas costs are community funded                                                  | Gas costs are managed <br/>by API3 using upgrade fees |

Reviewer: I feel like this table needs a leftmost row descriptor column e.g. for the first row "Available since", second row "Structure", third row "Deviation", etc.
